packages/itwin/reports-config-widget/src/test/Reports.test.tsx

```diff
@@ -169,27 +134,10 @@ describe("Reports View", () => {
 
   it("remove a report", async () => {
     const mockedReports: ReportCollection = reportsFactory();
-    const mockedReportsOriginalLength = mockedReports.reports.length;
-    server.use(
-      rest.get(
-        `${REPORTS_CONFIG_BASE_URL}/insights/reporting/reports`,
-        async (_req, res, ctx) => {
-          return res(ctx.delay(200), ctx.status(200), ctx.json(mockedReports));
-        }
-      ),
-      rest.delete(
-        `${REPORTS_CONFIG_BASE_URL}/insights/reporting/reports/${mockedReports.reports[0].id
-        }`,
-        async (_req, res, ctx) => {
-          mockedReports.reports = mockedReports.reports.filter(
-            (report) => report.id !== mockedReports.reports[0].id
-          );
-          return res(ctx.delay(100), ctx.status(204));
-        }
-      )
-    );
 
-    const { user } = render(<Reports />);
+    mockGetReports.mockReturnValue(mockedReports.reports);
```

Comment:
Is expecting to call `mockGetReports` twice the reason why we use `mockReturnValue` here instead of `mockReturnValueOnce` in other functions?

Reply:
Yes.